src/views/Benchmark/index.jsx

```diff
@@ -1,22 +1,21 @@
 import PropTypes from 'prop-types';
 import { Component } from 'react';
-import { withStyles } from '@material-ui/core/styles';
 import MetricsGraphics from 'react-metrics-graphics';
-
 import { curveLinear } from 'd3';
 import { subbenchmarksData } from '@mozilla-frontend-infra/perf-goggles';
+import { withRouter } from 'react-router-dom';
 import Header from '../../components/Header';
 import CONFIG from '../../config';
 import prepareData from '../../utils/prepareData';
 import './benchmark.css';
 
-const styles = () => ({
-  root: {},
-});
-
 class Benchmark extends Component {
```

Comment:
Following the rest of my review comments, export this directly and you can remove the need for `withRouter` since this component will be directly rendered by a `Route`:

```js
export default class Benchmark extends Component {
```